src/SwedbankPay.Sdk.Tests/PaymentTests/CardPaymentTests.cs

```diff
@@ -75,5 +75,13 @@ public async Task CreatePayment_ShouldReturnPayment()
             Assert.NotNull(creditCardPayment);
             Assert.Equal(paymentRequest.Payment.Metadata["key1"], creditCardPayment.PaymentResponse.Metadata["key1"]);
         }
+
+        [Fact]
+        public async Task CreateVerifyPaymentAndGetVerifications_ShouldReturnVerificationsWithVerificationList()
+        {
+            var paymentRequest = this.paymentRequestBuilder.WithCreditcardTestValues(this.payeeId, Operation.Verify).BuildCreditardPaymentRequest();
+            var creditCardPayment = await this.Sut.Payments.CardPayments.Create(paymentRequest, PaymentExpand.All);
+            var getCreditcardPayment = await this.Sut.Payments.CardPayments.Get(creditCardPayment.PaymentResponse.Id);
```

Comment:
Codacy found an issue: [Remove the unused local variable 'getCreditcardPayment'.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=5742807)